### ISSUE TITLE
addSubGrid does not work: TypeError: Failed to execute 'appendChild' on 'Node':

### ISSUE THREAD

**Issue by vmihailenco** [gridstack/gridstack.js#2553]
The documention has this section

![subgrid](https://github.com/gridstack/gridstack.js/assets/290976/db05461a-cd82-4256-8562-bc6f21cf9a52)

1. It does not seem to work: https://jsfiddle.net/3adz58bm/ (click on Add widget)

```shell
_display/?editor_console=true:142 Uncaught TypeError: Failed to execute 'appendChild' on 'Node': parameter 1 is not of type 'Node'.
    at addNewWidget (_display/?editor_console=true:142:9)
    at HTMLButtonElement.onclick (_display/?editor_console=true:119:34)
```

2. It mentions `makeSubgrid`, but uses `addSubGrid`.

4. Probably `makeSubgrid` has a typo and it should be `makeSubGrid`.

**Comment by adumesny** [comment]
you can't just add any div to a grid and not makeWidget() first, then figure out on to make content by a subgrid, which is a dup of #2553. closing.

**Comment by adumesny** [comment]
meant #2554 - reopening as at very least doc has to change. sorry...

**Comment by adumesny** [comment]
fixed in #2555
